cli/stat.go

```diff
@@ -72,6 +80,12 @@ func mainStat(ctx *cli.Context) error {
 		StatOpts: minio.StatObjectOptions{
 			ServerSideEncryption: sse,
 		},
+		ListExisting: ctx.Bool("list-existing"),
+		ListFlat:     ctx.Bool("list-flat"),
+		ListPrefix:   ctx.String("prefix"),
+	}
+	if b.ListExisting {
+		b.CreateObjects = 0
```

Comment:
"CreateObjects" acts as a limit in other `list-existing` implementations. This should be similar to that.